tests/plugins/grid_manipulation.test.ts

```diff
@@ -1021,9 +1021,15 @@ describe("Rows", () => {
       expect(model.getters.getRowSize(sheetId, 3)).toBe(10);
       expect(model.getters.getRowSize(sheetId, 4)).toBe(20);
       expect(model.getters.getRowSize(sheetId, 5)).toBe(size);
-      const dimensions = model.getters.getMainViewportRect();
-      expect(dimensions).toMatchObject({ width: 192, height: 124 });
       expect(model.getters.getNumberRows(sheetId)).toBe(6);
+      const dimensions = model.getters.getMainViewportRect();
+      expect(dimensions).toMatchObject({ width: 1000, height: 1000 });
+      model.dispatch("RESIZE_SHEETVIEW", {
+        width: DEFAULT_CELL_WIDTH,
+        height: DEFAULT_CELL_HEIGHT,
+      });
+      const newDimensions = model.getters.getMainViewportRect();
+      expect(newDimensions).toMatchObject({ width: 192, height: 124 });
```

Comment:
I'm gonna trust you on that, but why resizing the sheet to `{DEFAULT_CELL_WIDTH, DEFAULT_CELL_HEIGT}` leads to a size of `{192, 124}` ?